include/mbedtls/platform_util.h

```diff
@@ -55,6 +66,38 @@ extern "C" {
  */
 void mbedtls_platform_zeroize( void *buf, size_t len );
 
+#if defined(MBEDTLS_HAVE_TIME_DATE)
+/**
+ * \brief      Thread safe implementation of gmtime()
```

Comment:
This is really an implementation of gmtime_r(), not gmtime().

Reply:
Addressed in c52ef40 - please have a look.